libs/gltfio/include/gltfio/Animator.h

```diff
@@ -18,10 +18,14 @@
 #define GLTFIO_ANIMATOR_H
 
 #include <gltfio/FilamentAsset.h>
+#include <gltfio/FilamentInstance.h>
 
 namespace gltfio {
 
-namespace details { struct FFilamentAsset; }
+namespace details {
```

Comment:
in another PR, maybe we should get rid of details:: in gltfio too?

Reply:
Yes I would definitely like to do that.